tests/ed25519.rs

```diff
@@ -24,6 +24,8 @@ use sha2::Sha512;
 
 #[cfg(test)]
 mod vectors {
+    use ed25519::signature::Signature as _;
```

Comment:
This is needed to access the [`Signature::from_bytes`](https://docs.rs/signature/1.0.0-pre.5/signature/trait.Signature.html) method.

Elsewhere in this PR (e.g. in the docs) I've used `TryFrom`/`TryInto` to avoid this. Perhaps we should pick a single approach and standardize on it? (e.g. it seems nice if Keypair and PublicKey also had `TryFrom` impls)